client/a8c-for-agencies/sections/overview/sidebar/growth-accelerator/index.tsx

```diff
@@ -56,9 +69,7 @@ export default function OverviewSidebarGrowthAccelerator() {
 				<Button
 					className="overview__growth-accelerator-footer-schedule-call"
 					variant="primary"
-					href="https://savvycal.com/automattic-for-agencies/agency-success?utm_campaign=overview"
-					target="_blank"
-					rel="noopener noreferrer"
+					disabled={ isFetchingScheduleCallLink }
```

Comment:
Can we add isBusy here?

```suggestion
					disabled={ isFetchingScheduleCallLink }
					isBusy={ isFetchingScheduleCallLink }
```

Reply:
Good suggestion. Done this has been applied 👍